babel.config.js

```diff
@@ -1,20 +1,20 @@
 module.exports = function(api) {
   api.cache(true);
   return {
-    presets: ['babel-preset-expo'],
+    presets: ["babel-preset-expo"],
     plugins: [
       [
-        'module-resolver',
+        "module-resolver",
         {
-          root: ['./'],
+          root: ["./"],
           alias: {
-            Screens: './src/screens',
-            Store: './src/store',
-            Components: './src/components',
-            Lib: './src/lib',
+            Screens: "./src/screens",
+            Store: "./src/store",
+            Components: "./src/components",
+            Lib: "./src/lib"
```

Comment:
please leave the trailing comma, it makes it easier to add new items, and clutters the diff

Reply:
Sorry, I think that's my code formatter. I'll review the settings.